elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -24,9 +24,16 @@ class PfeContentSet extends PFElement {
   }
 
   get isTab() {
+    var breakpointValue;
+    if (this.hasAttribute('pfe-breakpoint')) {
+      breakpointValue = this.getAttributeNode('pfe-breakpoint').value;
+      breakpointValue = breakpointValue.replace(/\D/g,'');
+    } else {
+      breakpointValue = 768;
```

Comment:
Should the breakpoint be 767px? I'm use to tablet being 768 and it changes design at tablet. Or below could be `? this.parentNode.offsetWidth >= breakpointValue`?

Reply:
700 as a default approved in PFE touchpoints meeting 👍